starboard/extension/extension_test.cc

```diff
@@ -623,6 +624,26 @@ TEST(ExtensionTest, CobaltAccessibilityExtension) {
       << "Extension struct should be a singleton";
 }
 
+TEST(ExtensionTest, SocketReceiveMultiMsg) {
+  typedef CobaltExtensionSocketReceiveMultiMsgApi ExtensionApi;
+  const char* kExtensionName = kCobaltExtensionSocketReceiveMultiMsgName;
+
+  const ExtensionApi* extension_api =
+      static_cast<const ExtensionApi*>(SbSystemGetExtension(kExtensionName));
+  if (!extension_api) {
+    return;
+  }
+
+  EXPECT_STREQ(extension_api->name, kExtensionName);
+  EXPECT_EQ(extension_api->version, 1u);
+  EXPECT_NE(extension_api->MemoryMapFile, nullptr);
```

Comment:
change `MemoryMapFile` here - this breaks build. copy-paste error ?

Reply:
Yes. Thanks for catching